client/src/components/ReviewList.js

```diff
@@ -0,0 +1,23 @@
+// arranges the div element of all related reviews by mapping in to review tiles
+import React, { useState } from "react";
+import ReviewTile from "./ReviewTile.js";
+
+const ReviewList = (props) => {
+  let reviewTiles = ""
+  if(props.reviews) {
```

Comment:
are you conditionally rendering something else if there are no reviews? If not, then we should remove this `if` statement. `props.reviews` should be initialized as an empty array, meaning that if there are no reviews we will not get an error when trying to use `.map`

however, if you have conditional text for when there are no reviews to say "No reviews yet, leave a review or check back later" (whatever wording you want to use), then you can keep the if statement and include some messaging when there are no reviews